eunkyung/routes/likeRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const likeController = require('../controllers/likeController')
+const router = express.Router()
+
+
+
+router.post('', likeController.likePost)
+
+
+
+
+module.exports = { router }
```

Comment:
- 우선 불필요한 공백은 제거해주세요.
- Third-party-module을 import하는 부분과 Custom-module을 import 하는 부분을 구분해주면 가독성과 유지보수 측면에서 좋을 것 같습니다.
- 기본 엔드포인트에 공백 대신 '/'를 입력해주세요!
- 
```suggestion
const express = require('express')

const likeController = require('../controllers/likeController')
const router = express.Router()

router.post('/', likeController.likePost)

module.exports = { router }
```